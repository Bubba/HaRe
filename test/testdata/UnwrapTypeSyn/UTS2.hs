module UTS2 where

type Foo = (  Int,    String  )

type Bar = String

getInt :: Foo -> Int
getInt (i, _) = i
