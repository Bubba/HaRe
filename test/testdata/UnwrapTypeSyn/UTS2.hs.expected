module UTS2 where

type Bar = String

getInt :: (  Int,    String  ) -> Int
getInt (i, _) = i
